### PR TITLE
Mobile navigation

### PR DESCRIPTION
**Pull request by Fraasi**
As with all my pull requests, I'm not sure if this is the best/most efficient way of doing this, but it works & looks nice 😄   
It was easier for me to write plain css to wrap my head around the effects, so I will 'styled-component' it later.

### CODE REVIEW

**Comment by Fraasi** [comment]
Ok @meiamsome, this is all done and ready to merge after your review.  
It's all styled component now. It might be a bit resource heavy for old mobile phones as it is adding/removing quite a lot class names on click for the effects, but it works nicely and looks sweet 😄 .

**Review by gruselhaus — APPROVED** [review]

**Comment by Fraasi** [comment]
@gruselhaus thanks for approving 😄 but there is still a lot to improve on this. It is a working example, but I am working with meiamsome to make this much better.

File: src/components/Header/index.tsx [hunk 1]
```diff
@@ -1,18 +1,11 @@
 import React, { Component } from 'react';
-import { StyledHeader, Nav } from './index.styles';
+import { StyledHeader } from './index.styles';
 
 export default class Header extends Component {
   render() {
     return (
       <StyledHeader className="header">
```
Comment (meiamsome) [inline src/components/Header/index.tsx]: This should be self-closing now. Also, drop the `className`.

File: src/components/MobileNavigation/index.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+      padding: 10px 20px 10px 20px;
+      background-color: #2494c1;
+      border-radius: 5px;
+      transition: all ease-in-out 0.5s;
+    }
+
+    & .nav-link.open {
+      left: 0;
+    }
+  `};
```
Comment (meiamsome) [inline src/components/MobileNavigation/index.styles.tsx]: This semicolon is what's causing the lint problems

File: src/components/MobileNavigation/index.tsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+import { Link } from 'react-router-dom';
+// import Button from '../Button';
+import { Backdrop, BurgerContainer, Nav } from './index.styles';
+
+export default class BurgerIcon extends Component<any, any> {
+  pageWrapper: any;
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      open: '',
```
Comment (meiamsome) [inline src/components/MobileNavigation/index.tsx]: Prefer a boolean here

File: src/components/MobileNavigation/index.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        >
+          <div id="burger">
+            <span>&nbsp;</span>
+            <span>&nbsp;</span>
+            <span>&nbsp;</span>
+          </div>
+        </BurgerContainer>
+
+        <Nav
+          className={`nav-links ${open}`}
```
Comment (meiamsome) [inline src/components/MobileNavigation/index.tsx]: I don't think there should ever be a need for `className` with styled-components. Prefer `open={open}`

**Pull request merged**